spec/models/authority_spec.rb

```diff
@@ -3,11 +3,14 @@
 describe Authority do
   describe "validations" do
     it 'should ensure a unique short_name' do
-      @a1 = create(:authority, short_name: "Existing")
-      @a2 = build(:authority, short_name: "Existing")
+      existing_authority = create(:authority, short_name: "Existing")
+      new_authority = build(:authority, short_name: "Existing")
 
-      expect(@a2.valid?).to eq false
-      expect(@a2.errors.messages[:short_name]).to eq(["has already been taken"])
+
+      expect(existing_authority.valid?).to eq true
```

Comment:
Isn't this test superfluous because if `create` succeeds on line 6 then this will always be true? In other words it's not possible for this to fail.

I think you could remove this and not assign the first Authority, and the test would be clear and focused.